codegen/facelift/templates/IPCProxyAdapter.template.cpp

```diff
@@ -134,7 +187,7 @@ void {{className}}::deserializeSignal(InputIPCMessage &msg)
 void {{className}}::set{{property}}({{property.cppMethodArgumentType}} newValue)
 {
     {% if (not property.type.is_interface) %}
-    ipc()->sendSetterCall(memberID(MethodID::set{{property.name}}, "set{{property.name}}"), newValue);
+    ipc()->sendSetterCall("{{property.name}}", newValue);
```

Comment:
FYI, the construct you broke here is meant to provide a way for that generated code to be usable with both an IPC transport channel where method IDs are strings (such as DBus), and an IPC where method IDs are integer (such as the the current "local" IPC, and the hi-performance IPC which I have been working on). Using integers instead of strings is obviously more efficient.
Your change will make the introduction of a new IPC harder.

Reply:
Can we please stop the "performance" discussion? now you are discussing about "future possible performance degrades with some not in-place high performance IPC!!!".

Reply:
The local IPC uses integer IDs, so we are not talking about the future.. The current approach should work with your changes.

Reply:
The performance claims needs to be backed up by actuall KPI measurement
There "Local" version AFAIK uses internal Qt signal/slots so we can't have seriously a discussion over difference of treating a signal with integer and a signal with string!

Reply:
I have no doubt that sending an integer (which does not even need to be wider than 8 bits) is faster than sending a multi-character string. Also, string comparisons are not free, so the processing of an incoming request is also faster if an integer is used as method ID since no comparison is even needed.

Reply:
A simple benchmark of signal 30 character long being emitted/received 10000 times shows followings:
https://github.com/idleroamer/facelift/tree/feature/benchmark_local_signals

Current implementation
Signal emission/receive duration: 1463 ms

Current PR: 
Signal emission/receive duration:1102 ms

A signal name comparison is not a bottleneck.

Reply:
Do those figures tell us that using a string is faster than using an integer ?
The reason why integers can be used is not really because using strings has been identified as a performance hotspot, but rather that it is necessary for IPCs which use integers as messageID. I do not understand why your change has to break that.

Reply:
No, those figures tell us whether we send MessageIDs as strings or as integers is not a performance bottleneck.
It as well tells us performance arguments without benchmarks has no values.

I appreciate your time but I have the feeling you even haven't looked at the PR? the "sendSetterCall" is internal call between IPCProxy and Local/DBusIPCProxyBinder and using integers there is simply superfluous,

You had performance claims now you are talking again about some IPC which is not even there yet AGAIN.
Please let me know if you personally hold this PR against your views/plan for facelift so I don't take your time anymore.

Reply:
There is a misunderstanding here.
How about having a call tomorrow afternoon ? I guess we could use google meet for that.

Reply:
That is a fine suggestion. Though tomorrow is a little tight in the afternoon, so let me know if tomorrow morning or the Friday's afternoon fits your calendar, otherwise we postpone to next week.